src/pkg/utils/io.go

```diff
@@ -5,44 +5,24 @@
 package utils
 
 import (
-	"archive/tar"
-	"bufio"
-	"crypto/sha256"
 	"fmt"
-	"io"
-	"io/fs"
-	"net/http"
 	"os"
 	"os/exec"
 	"path/filepath"
-	"regexp"
-	"strings"
-	"time"
 
 	"github.com/defenseunicorns/zarf/src/config"
 	"github.com/defenseunicorns/zarf/src/pkg/message"
 	"github.com/defenseunicorns/zarf/src/pkg/utils/helpers"
-	"github.com/defenseunicorns/zarf/src/types"
-	"github.com/otiai10/copy"
 )
 
 const (
-	dotCharacter  = 46
 	tmpPathPrefix = "zarf-"
 )
 
-// TextTemplate represents a value to be templated into a text file.
-type TextTemplate struct {
-	Sensitive  bool
-	AutoIndent bool
-	Type       types.VariableType
-	Value      string
-}
-
 // MakeTempDir creates a temp directory with the zarf- prefix.
 func MakeTempDir(basePath string) (string, error) {
 	if basePath != "" {
-		if err := CreateDirectory(basePath, 0700); err != nil {
+		if err := helpers.CreateDirectory(basePath, 0700); err != nil {
 			return "", err
 		}
 	}
```

Comment:
`MakeTempDir` can prob be brought over to `helpers` (unless the internal debug is what made you keep it here)

Reply:
It was also the temp path prefix - IMO it is relatively Zarf specific - os.MkdirTemp is there for folks that want an OOB experience.